cclient/api/core/add_neighbors.h

```diff
@@ -26,19 +26,20 @@ extern "C" {
 #endif
 
 /**
- * @brief
+ * @brief Add a list of neighbors to the IRI node.
+ *
+ * Assumes <b>addNeighbors</b> command is abaliable on the node. It should be noted that this is only temporary, and the
+ * added neighbors will be removed from your set of neighbors after you relaunch IRI.
+ *
+ * The URI format:
+ *
+ * - udp://IPADDRESS:PORT
+ * - tcp://IPADDRESS:PORT
  *
- * Add a list of neighbors to iri node. It should be noted that this is only
- * temporary, and the added neighbors will be removed from your set of neighbors
- * after you relaunch IRI.
- *
- * @param service client service
- * @param req Request containing list of neighbors to add
- * @param res Response containing a number of neighbor
- * @return
- *  - `#RC_OK`
- *  - `#RC_CCLIENT_OOM`
- *  - `#RC_ERROR`
+ * @param[in] service client service
+ * @param[in] req A list of URIs.
+ * @param[out] res Number of neighbors that were added.
```

Comment:
what if result will contain more info in the future?
maybe: "res Response from the node, containing the number of neighbors that were added." ?

Reply:
Responses are all from the connected node in Core APIs, 
https://github.com/iotaledger/entangled/pull/1093/files#diff-8ef0c618acc3cbe03ab59dee66f2a621R14